### PR TITLE
enable samba at wizard

### PR DESCRIPTION
**Pull request by CvH**
set the samba default to enabled at the wizard

### CODE REVIEW

File: src/resources/lib/modules/services.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ def load_values(self):
             # SAMBA
 
             if os.path.isfile(self.SAMBA_NMDB) and os.path.isfile(self.SAMBA_SMDB):
-                self.struct['samba']['settings']['samba_autostart']['value'] = self.oe.get_service_state('samba')
+                self.struct['samba']['settings']['samba_autostart']['value'] = "1"
```
Comment (MilhouseVH) [inline src/resources/lib/modules/services.py]: It doesn't make any difference to Python in this case, but the convention in this file is to use apostrophes for strings, would you mind changing `"1"` to `'1'`?

**Comment by CvH** [comment]
done

**Comment by MilhouseVH** [comment]
There's a problem with this PR... once Samba is enabled, it can't be disabled.

The following patch works (this PR needs to be reverted) and allows Samba to be disabled during and after the wizard:

http://sprunge.us/cJLi

**Comment by CvH** [comment]
@MilhouseVH fixed

File: src/resources/lib/modules/services.py [hunk 2]
```diff
@@ -305,7 +305,7 @@ def load_values(self):
             # SAMBA
 
             if os.path.isfile(self.SAMBA_NMDB) and os.path.isfile(self.SAMBA_SMDB):
-                self.struct['samba']['settings']['samba_autostart']['value'] = self.oe.get_service_state('samba')
+                self.struct['samba']['settings']['samba_autostart']['value'] = '1'
```
Comment (MilhouseVH) [inline src/resources/lib/modules/services.py]: This needs to be reverted back to how it was - line 388 should be:
```
self.struct['samba']['settings']['samba_autostart']['value'] = self.oe.get_service_state('samba')
```

**Comment by CvH** [comment]
@MilhouseVH changed :) (also edit the author)

**Pull request merged**